tests/table/dynamic_table_plugin.test.ts

```diff
@@ -0,0 +1,222 @@
+import { Model } from "../../src";
+import { toZone, zoneToXc } from "../../src/helpers";
+import { TABLE_PRESETS } from "../../src/helpers/table_presets";
+import { UID } from "../../src/types";
+import {
+  copy,
+  createDynamicTable,
+  createTable,
+  cut,
+  deleteTable,
+  paste,
+  setCellContent,
+  updateFilter,
+  updateTableConfig,
+  updateTableZone,
+} from "../test_helpers/commands_helpers";
+import { TABLE_STYLE_ALL_RED } from "../test_helpers/constants";
+import { getCell } from "../test_helpers/getters_helpers";
+import {
+  getExportedExcelData,
+  getFilterHiddenValues,
+  toCellPosition,
+} from "../test_helpers/helpers";
+
+let model: Model;
+let sheetId: UID;
+
+const oldTablePresets = { ...TABLE_PRESETS };
+
+beforeEach(() => {
+  model = new Model();
+  sheetId = model.getters.getActiveSheetId();
+  TABLE_PRESETS.TestStyleAllRed = TABLE_STYLE_ALL_RED;
+});
+
+afterEach(() => {
+  Object.keys(TABLE_PRESETS).forEach((key) => delete TABLE_PRESETS[key]);
+  Object.assign(TABLE_PRESETS, oldTablePresets);
+});
+
+function getTables(model: Model, sheetId: UID) {
+  return model.getters
+    .getTables(sheetId)
+    .map((table) => ({ ...table, zone: zoneToXc(table.range.zone) }));
+}
+
+describe("Dynamic tables", () => {
```

Comment:
add a test which duplicates a sheet with a dynamic table.  It doesn't seem there is one.

Reply:
And it was, in fact, bugged :sweat_smile: